### ISSUE TITLE
Error with keys using Promise.all

### ISSUE THREAD

**Issue by ivanhuay** [ramiel/mongoose-sequence#23]
I'm trying to save multiple records in a single step but it ends with an error

* model:
```javascript
...
let personSchema= new mongoose.Schema({
    _id: Number,
    name: {
        type:     String,
        required: true
    }
});
personSchema.plugin(AutoIncrement);

module.exports = mongoose.model('Person', personSchema);

```
* Usage
```javascript
function createMultipleUsers(){
    return Promise.all([
        Person.create({name:'ivan'}),
        Person.create({name:'axel'})
   ]);
}
createMultiple();
```
* Error:
```
E11000 duplicate key error collection: db.counters index: id_1_reference_value_1 dup key: { : "_id", : null }
```

**Comment by ramiel** [comment]
Can you try to pass the option {_id:false} to the schema options, as explained in the readme?

**Comment by ivanhuay** [comment]
@ramiel  we tested it with _id: false in the schema and it did not work. Returns the same error

**Comment by ramiel** [comment]
I'll look at this soon

**Comment by ramiel** [comment]
Ok, I can reproduce the error. I'll be back soon

**Comment by ramiel** [comment]
Perfect, I found the issue. I'll push a new version today

**Comment by ramiel** [comment]
Published in version 4.0.1
